src/components/ItemTop.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import searchIcon from "../assets/ic_search@3x.png";
+import dropdownIconPC from "../assets/ic_dropdown@3x.png";
+import dropdownIconMobile from "../assets/ic_dropdown_mobile@3x.png";
+import "../styles/items.css";
+
+const H1 = ({ children }) => {
+  return <h1 className="h1__items">{children}</h1>;
+};
+
+const Search = () => {
+  return (
+    <form className="search">
+      <button className="search__button">
+        <img src={searchIcon} alt="search" width="15px" height="15px" />
+      </button>
+      <input
+        className="search__input"
+        placeholder="검색할 상품을 입력해주세요"
+      />
+    </form>
+  );
+};
+
+const LinkBtn = ({ children, to }) => {
+  return (
+    <Link to={to}>
+      <button className="regist">{children}</button>
+    </Link>
+  );
+};
+
+const DropDown = ({ filterName, filter, setFilter }) => {
+  const [dropdown, setDropdown] = useState(false);
```

Comment:
```suggestion
const [isDropdownOpen,setIsDropdownOpen] = useState(false)
```
일반적으로 boolean 상태를 다룰 때는 의문문 형태의 state로 이름짓곤 합니다.
예를들어, boolean타입은 dropdown이 열리거나, 닫히거나 둘중 하나입니다.
그래서 dropdown은 상태를 명확히 알기 어렵기 때문에
```javascript 
const [isDropdownOpen,setIsDropdownOpen] = useState(false)
```
의 방식이 구체적일 것 같습니다.

이름 짓기가 어렵다면 이 [블로그](https://www.rldnrl.dev/blog/naming-cheatsheet)를 참고 해 보세요

Reply:
아직은 익숙하지가 않네요. 의식하지 않고 만들 수 있을 만큼 익숙해져야할 것 같습니다